src/api/app/views/webui2/shared/_group_members.html.haml

```diff
@@ -0,0 +1,20 @@
+- if policy(group).update?
+  %ul.list-inline
+    %li.list-inline-item
+      = link_to(group_edit_title_path(title: group.title), class: 'nav-link') do
+        %i.fas.fa-edit.text-secondary
+        Edit Group
+
+- if group.users.none?
```

Comment:
[nitpick] using `blank?` instead saves as a query to the database as it is already cached for `group.users`